SWLOR.Game.Server/Service/GuiService/GuiWindowType.cs

```diff
@@ -50,5 +50,6 @@ public enum GuiWindowType
         DroidAI = 45,
         Currencies = 46,
         StatRebuild = 47,
+        HoloNet = 48,
```

Comment:
Can you bump this up to 49? I used the same ID for another window accidentally. Sorry

Reply:
Bumped to 49